src/browser/api/application.js

```diff
@@ -707,6 +693,17 @@ function run(identity, mainWindowOpts, userAppConfigArgs) {
     coreState.setAppRunningState(uuid, true);
 
     ofEvents.emit(route.application('started', uuid), { topic: 'application', type: 'started', uuid });
+
+    const { preloadScripts } = mainWindowOpts;
+    const loadUrl = () => app.mainWindow.loadURL(app._options.url);
+
+    if (appWasAlreadyRunning) {
+        loadUrl();
+    } else {
+        System.downloadPreloadScripts(windowIdentity, preloadScripts)
+            .catch(() => {})
```

Comment:
maybe a log? we really dont care?

Reply:
We will catch most probable errors elsewhere. There was no logging here before, but I can add some generic error logging here no problem